library/src/hcc_detail/hipblas.cpp

```diff
@@ -14239,6 +14239,8 @@ hipblasStatus_t hipblasZgemmStridedBatched(hipblasHandle_t             handle,
 }
 
 // gemm_ex
+// First, we are querying the intended int8 layout from rocBLAS and passing in the result.
```

Comment:
This comment needs some cleanup to improve clarity.
Can you change: "Note for int8 users - please read rocblas_gemm_ex documentation on int8 data layout requirements. hipBLAS makes the assumption that the data layout is in the preferred format for a given device as documented in rocBLAS."